pkg/controller/volume/pvcprotection/pvc_protection_controller.go

```diff
@@ -274,13 +292,48 @@ func (c *Controller) askAPIServer(pvc *v1.PersistentVolumeClaim) (bool, error) {
 	return false, nil
 }
 
-func podUsesPVC(pod *v1.Pod, pvc string) bool {
+func (c *Controller) podUsesPVC(pod *v1.Pod, pvc *v1.PersistentVolumeClaim) bool {
+	// The following text is based on how pod shutdown was
+	// initially described to me. During PR review, @msau42 pointed out
```

Comment:
Prefer not to put my name into the comment. Linking to an issue # to track is fine.

Reply:
Okay. It was already part of the issues's TODO list, so I am linking to that now.

I also got more paranoid about breaking something and therefore moved the invocation of `podIsShutDown`: instead of being used as a general shortcut for all volume types, it is now only used for ephemeral volumes. So even with the alpha feature enabled, nothing changes for PVCs.